snappy_pipeline/workflows/cbioportal_export/__init__.py

```diff
@@ -11,196 +11,253 @@
+CNA_DATA_FILES = ["data_cna_gistic.txt", "data_cna_log2.txt", "data_segment.txt"]
+
+# case lists files
+CASE_LIST_FILES = {
+    "sequenced": {
+        "filename": "all_cases_with_mutation_data.txt",
+        "name": "Sequenced tumors",
+        "description": "Tumors with somatic variant calls",
+        "stable_id": "sequenced",
+        "category": "all_cases_with_mutation_data",
+    },
+    "cna": {
+        "filename": "all_cases_with_cna_data.txt",
+        "name": "Sequenced tumors",
+        "description": "Tumors with somatic Copy Number Alteration calls",
+        "stable_id": "cna",
+        "category": "all_cases_with_cna_data",
+    },
+    "rna_seq_mrna": {
+        "filename": "all_cases_with_mrna_rnaseq_data.txt",
+        "name": "Tumors with expression data",
+        "description": "Tumors with mRNA seq expression data",
+        "stable_id": "rna_seq_mrna",
+        "category": "all_cases_with_mrna_rnaseq_data",
+    },
+    "cnaseq": {
+        "filename": "all_cases_with_mutation_and_cna_data.txt",
+        "name": "Sequenced tumors",
+        "description": "Tumors with somatic variant & CNA calls",
+        "stable_id": "cnaseq",
+        "category": "all_cases_with_mutation_and_cna_data",
+    },
+    "3way_complete": {
+        "filename": "all_cases_with_mutation_and_cna_and_mrna_data.txt",
+        "name": "Sequenced tumors with expression",
+        "description": "Tumors with somatic variant calls, CNA calls & expression data",
+        "stable_id": "3way_complete",
+        "category": "all_cases_with_mutation_and_cna_and_mrna_data",
+    },
+}
 
 
 DEFAULT_CONFIG = r"""
 step_config:
   cbioportal_export:
-    # Paths to snappy steps containing results to be uploaded
-    path_ngs_mapping: ../ngs_mapping                                        # REQUIRED
-    path_gene_expression_quantification: ../gene_expression_quantification  # Set to '' in absence of mRNA_seq
-    path_somatic_variant_filtration: ../somatic_variant_filtration          # REQUIRED
-    path_copy_number_step: ../somatic_targeted_seq_cnv_calling              # Set to '' for panels, and change for WGS
-    # Select tools & filter set
-    cnv_tool: copywriter                                                    # Other option: cnvkit, Control_FREEC (unsupported)
-    tools_somatic_variant_calling: [ "mutect" ]                             # Possibly scalpel, mutect2, strelka2
-    filter_set: dkfz_only                                                   # Possibly dkfz_and_ebfilter, dkfz_and_ebfilter_and_oxog, ...
-    exclude_variant_with_flag: LowFisherScore  # REQUIRED
-    # Additional parameters
-    vep_data_path: REQUIRED   # Variant Effect Predictor DB for vcf -> maf conversion, must match genome release
-    filter_vcf: REQUIRED      # Common germline variants for vcf -> maf conversion, must match genome release
-    ncbi_build: GRCh37
+    # Required for RNA expression
+    path_ngs_mapping: ""
```

Comment:
Why not set the path to "../ngs_mapping"?

Reply:
The path is used to decide whether or not to include expression data in the cBioPortal output. The same is done with `path_copy_number` & (to a lesser extent) `path_somatic_variant`.